.github/workflows/codecov.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: 17
+          java-version: 19
```

Comment:
@hizmailovich these changes should be located in the separate pr

Reply:
Updated java-version from pom.xml, works fine with java 17